packages/@okta/vuepress-theme-prose/global-components/EventTypes.vue

```diff
@@ -39,7 +39,9 @@
       SmartLink: () => import("../components/SmartLink"),
     },
     created() {
-      this.eventTypes = eventTypes.versions[1].eventTypes.filter(eventType => !eventType.beta)
+      this.eventTypes = eventTypes.versions
+        .find(version => version.version == "V2").eventTypes
```

Comment:
We are planning on removing v1 event types from this very large file since it is never used.